### PR TITLE
fix(Communities): Fix channels reordering

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #6292

### What does the PR do

Add `reorder` method to `SubModel` and remove prepended communityID to chatID.

### Affected areas

Communities

### CODE REVIEW

**Comment by elina2015** [comment]
@borismelnik hey, the owner reordered the channels and see them like this:
image
But the member of the community can see the channels by another way:
image

Could you kindly check?

File: src/app_service/service/community/service.nim [hunk 1]
```diff
@@ -804,7 +804,7 @@ QtObject:
               self.joinedCommunities[communityId].chats[prev_chat_idx].position = chat.position
               chatDetails.updateMissingFields(self.joinedCommunities[communityId].chats[prev_chat_idx])
               self.chatService.updateOrAddChat(chatDetails) # we have to update chats stored in the chat service.
-              self.events.emit(SIGNAL_COMMUNITY_CHANNEL_REORDERED, CommunityChatOrderArgs(communityId: updatedCommunity.id, chatId: fullChatId, categoryId: chat.categoryId, position: chat.position))
+              self.events.emit(SIGNAL_COMMUNITY_CHANNEL_REORDERED, CommunityChatOrderArgs(communityId: updatedCommunity.id, chatId: chat.id, categoryId: chat.categoryId, position: chat.position))
```
Comment (micieslak) [inline src/app_service/service/community/service.nim:807]: `fullChatId` doesn't work with `self.chatService.getChatById`, plain `chat.Id` is expected as well. When `fullChatId` is used, chat details is not found and chatService is not updated. Error is msg is emitted to the output as well.

**Review by micieslak — CHANGES_REQUESTED** [review]

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Comment by micieslak** [comment]
resolved in https://github.com/status-im/status-desktop/pull/6840

**Pull request closed without merging**